src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -733,10 +734,22 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		};
 	}
 
-	private rebuildActionBar() {
+	public rebuildActionBar() {
 		let actions = this.getCurrentActions();
 		this.actionBar.clear();
-		this.actionBar.push(actions, { icon: true, label: false });
+		if (this.options.displayActions ?? true) {
```

Comment:
`this.options.displayActions` should be initialized default value, you shouldn't need to add `?? true`

Reply:
Moved the action bar changes its own PR: https://github.com/microsoft/azuredatastudio/pull/16178